tests/foreman/ui/test_dashboard.py

```diff
@@ -252,21 +253,23 @@ def test_positive_user_access_with_host_filter(
     with Session(test_name, user=user_login, password=user_password) as session:
         assert session.dashboard.read('HostConfigurationStatus')['total_count'] == 0
         assert len(session.dashboard.read('LatestErrata')['erratas']) == 0
-        repos_collection.setup_content(org.id, lce.id, upload_manifest=True)
+        rhel_contenthost.add_rex_key(target_sat)
+        repos_collection.setup_content(org.id, lce.id, upload_manifest=False)
```

Comment:
This is already False in the `setup_content` method
```suggestion
        repos_collection.setup_content(org.id, lce.id)
```